.github/workflows/build.yml

```
@@ -71,11 +71,25 @@ jobs:
           cache: "pip"
       - *install-emake
       - name: Run tests
-        run: make test
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:14:6: Quote this to prevent word splitting

(shellcheck)